app/src/androidTest/java/org/mozilla/focus/activity/DefaultBrowserTest.java

```diff
@@ -126,6 +126,11 @@ public void changeDefaultBrowser_whenChromeIsDefault() {
         // Click on "Default Browser" setting, this will brings up the Android system setting
         clickDefaultBrowserSetting(prefName);
 
+        // Click on the positive button of the tutorial dialog to continue
```

Comment:
Normally if the tutorial is not critical to the user or will change over time, I'll suggest we just skip it in the UI test
For example, change the preference before we launch the UI test.
But since we didn't use preference, instead we use PackageManager to check, let's keep it as is.